cns/service/main.go

```diff
@@ -546,6 +559,11 @@ func main() {
 		return
 	}
 
+	nmagentClients := restserver.NmagentMultiClient{
```

Comment:
maybe we can have a call to understand why we need this. Let's discuss these comments in tomorrow's sync